Gemfile

```diff
@@ -6,4 +6,10 @@ git_source(:github) { |repo| "https://github.com/#{repo}" }
 
 gemspec
 
-eval_gemfile("Gemfile.devel") rescue nil
+begin
+  eval_gemfile("Gemfile.devel")
+rescue StandardError
+  nil
+end
+
+gem "ogc-gml", github: "lutaml/ogc-gml"
```

Comment:
@kwkwan please move this to the gemspec

Reply:
@ronaldtse The gem `ogc-gml` has not been released yet.